src/app/shared/cms/components/cms-static-page/cms-static-page.component.html

```diff
@@ -1,11 +1,14 @@
 <div class="content-page">
   <div class="row">
+    <ish-error-message [error]="contentPageTreeError$ | async"></ish-error-message>
```

Comment:
Why would we need error handling here?
Maybe loading yes, but I am not sure about the error.